doczrc.js

```diff
@@ -11,7 +11,19 @@ export default {
     colors: {
       primary: RAW_COLORS.blue[700],
     },
+    styles: {
```

Comment:
This fixes that issue with xsmall btn commented earlier, kinda hacky though :/ Not really sure how to do a better theme override.